python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4060,7 +4081,9 @@ def test_isin_dataframe(data, values):
             got = gdf.isin(values)
```

Comment:
Can we pull the cudf running out of the try except? We only want to catch Pandas failing here, right?